### PR TITLE
initial version

### PR DESCRIPTION
**Pull request by fceller**

### CODE REVIEW

File: debugging/tasks/cleanout-server.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  // cleanout server
+  const data = {server: shortName};
+  let res = helper.httpWrapper('POST', '/_admin/cluster/cleanOutServer', data);
+
+  if (res.code !== 202) {
+    helper.fatal("cleanout failed: ", JSON.stringify(res));
+  }
+
+  const jobId = res.id;
+  print(res);
```
Comment (fceller) [inline debugging/tasks/cleanout-server.js:74]: @kvahed @neunhoef what to do with the jobId? How can I check it?

File: debugging/helper-cleanout-server.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  // show a server list
+  helper.showServers(conf, helper.getAgencyConfiguration());
+
+  // find a healthy coordinator
+  const cord = _.find(health, function (o, id) {
+    return o.Status === 'GOOD' && id.substring(0, 4) === 'CRDN';
+  });
+
+  if (!cord) {
+    helper.fatal("cannot find a health coordinator");
```
Comment (kvahed) [inline debugging/helper-cleanout-server.js:44]: typo: healthy

File: debugging/tasks/clear-cleanout-server.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    helper.fatal("Unknown server '" + name + "'");
+  }
+
+  const data = [[]];
+  const obj = {};
+
+  const k = '/arango/Target/CleanedServers';
+  obj[k] = { op: "erase", val: serverId };
+
+  data[0].push(obj);
```
Comment (kvahed) [inline debugging/tasks/clear-cleanout-server.js:44]: `data[0][0]`, right?

Comment (fceller) [reply debugging/tasks/clear-cleanout-server.js:44]: @kvahed I think "date[0].push(obj)" will add obj to data[0][0]. Is that not correct?

Comment (fceller) [reply debugging/tasks/clear-cleanout-server.js:44]: see clear-maintenance.js

Comment (kvahed) [reply debugging/tasks/clear-cleanout-server.js:44]: you are right

**Pull request merged**